Core/ExpectedException/ExpectedExceptionMethodBodyTransformer.cs

```diff
@@ -1,4 +1,4 @@
-﻿#region copyright
+#region copyright

```

Comment:
Changed file encoding?

Reply:
seems like a bug, i don't see any changes when comparing in a hex editor